com.unity.multiplayer.mlapi/Runtime/Core/NetworkBehaviour.cs

```diff
@@ -631,6 +633,7 @@ private void NetworkVariableUpdate(ulong clientId, int behaviourIndex)
 
                                             writer.WriteUInt16Packed((ushort)varBuffer.Length);
                                             buffer.CopyFrom(varBuffer);
+                                            deltaBuffer = varBuffer.Length;
```

Comment:
We should very that this is correct (and tested corectly)

Reply:
Refer to the pattern in NetworkObject.cs:293 where we compare prevLength to buffer.Length, relies on fewer assumptions in the code to get an accurate length. We should move to be consistent.